### ISSUE TITLE
monerod: [windows] Mysterious "disk leak," while Monero daemon is running

### ISSUE THREAD

**Issue by opportunistic** [monero-project/monero#8742]
This is driving me absolutely crazy, whenever I have monerod.exe running (after fully syncing), Windows begins to report disk free space decreasing at a pretty alarming rate, around 15GB every hour.  However, there is are not actually any new data being written to the disk...

WinDirStat reports the proper free space after fully scanning all files, and Windows only recognizes that this space is not actually allocated to any files after a full restart.

Something potentially worth noting is that this is being performed on an SSD with the operating system installed on it and also Veracrypt full disk encryption.

Please let me know if anyone has a solution or if more information is needed, this is driving me absolutely crazy.

**Comment by selsta** [comment]
Could you post your monerod config?

**Comment by opportunistic** [comment]
> Could you post your monerod config?

I do not believe I have one, I'm using Monero GUI wallet and I have not changed anything besides `--out-peers=512`

**Comment by selsta** [comment]
I'd recommend against setting such a high number of out peers but that's unrelated to your issue.

We have never received a similar report before so I suspect it's related to Veracrypt. Can you copy the blockchain to an external unencrypted drive and see if you can reproduce the issue?

**Comment by hyc** [comment]
> copy the blockchain to an external unencrypted drive

Move the entire bitmonero folder and tell monerod where to find it. I'm suspecting logfile activity, not database activity.

**Comment by opportunistic** [comment]
> > copy the blockchain to an external unencrypted drive
> 
> Move the entire bitmonero folder and tell monerod where to find it. I'm suspecting logfile activity, not database activity.

Watching the process with Process Monitor, there's not really any log file activity... mostly just ReadFile and WriteFile on the database.

**Comment by opportunistic** [comment]
Update!

After disabling this option on my blockchain folder (Windows seems to turn it on automatically sometimes), it seems like the "leak" has stopped!

![image](https://user-images.githubusercontent.com/109391343/219443146-60b2f6a2-32cf-46ba-a967-398d4664209e.png)

I'm not sure why this interaction occurs, would be worth fixing or warning the user about, though.

**Comment by selsta** [comment]
@hyc do you know anything about how this Windows option interacts with LMDB?
